.github/workflows/ci.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       matrix:
         NXF_VER:
           - "23.04.0"
+          - "24.10.3"
```

Comment:
Adding tests against the exact version of Nextflow we are currently using in GSP.